app/views/policies/_list_description.html.erb

```diff
@@ -1,7 +1,7 @@
 <ol class="policies document-list">
   <% policies.each do |policy| %>
     <%= content_tag_for(:li, policy, class: "document-row") do %>
```

Comment:
All of these `li` tags will have the same `id` of `"new_future_policy"` as by default, `ActiveModel::Model` returns `false` for `persisted?`. You'll have to override `persisted?` on `Future::Policy` and also make sure each instance returns a unique `id`.

Alternatively simply use `content_tag` and change the tests so that they verify the existence of the content without being wedded so tightly to the markup. Then non of the changes to `Future::Policy` will be necessary and you can simply pass in the rummager results, something like this: https://github.com/alphagov/whitehall/commit/1d4d5d10ff17f2682cd253b246b631ede01d8d90